apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/DMEClaimTransformerV2.java

```diff
@@ -454,6 +454,6 @@ static void addCodeExtension(
    */
   static void addDecimalExtension(
       ExplanationOfBenefit eob, CcwCodebookVariable ccwVariable, Optional<BigDecimal> optVal) {
-    eob.addExtension(TransformerUtilsV2.createExtensionDate(ccwVariable, optVal));
+    eob.addExtension(TransformerUtilsV2.createExtensionDate(ccwVariable, optVal.get()));
```

Comment:
we cannot do an optional get() without checking ifPresent() first. If this method does not handle empty values it should not accept an optional

Reply:
fixed